src/deploy/hasher_segments.js

```diff
@@ -22,13 +22,14 @@ const hasherCtor = ({ concurrentHash, hashAlgorithm = 'sha1' }) => {
 }
 
 // Inject normalized file names into normalizedPath and assetType
-const fileNormalizerCtor = ({ assetType = 'file' }) =>
+const fileNormalizerCtor = ({ assetType }) =>
   map((fileObj) => ({ ...fileObj, assetType, normalizedPath: normalizePath(fileObj.relname) }))
 
 // A writable stream segment ctor that normalizes file paths, and writes shaMap's
 const manifestCollectorCtor = (filesObj, shaMap, { statusCb, assetType }) => {
   if (!statusCb || !assetType) throw new Error('Missing required options')
   return objWriter((fileObj, _, cb) => {
+    // eslint-disable-next-line no-param-reassign
     filesObj[fileObj.normalizedPath] = fileObj.hash
```

Comment:
Ideally, we might want to return a copy, but in order to prevent any breaking, this keeps the fact that this function mutates arguments passed by its caller.